src/authentication/authentication.controller.ts

```diff
@@ -0,0 +1,98 @@
+import * as bcrypt from "bcrypt"
+import * as express from "express"
```

Comment:
```suggestion
import { Router, Request, Response } from "express"
```
itd.
wtedy nie musisz za każdym razem pisać `express.`

Reply:
zrobione